pkg/cmd/admin/registry/registry.go

```diff
@@ -254,6 +259,12 @@ func RunCmdRegistry(f *clientcmd.Factory, cmd *cobra.Command, out io.Writer, cfg
 		livenessProbe := generateLivenessProbeConfig(healthzPort)
 		readinessProbe := generateReadinessProbeConfig(healthzPort)
 
+		var secretBytes [randomSecretSize]byte
+		if _, err := cryptorand.Read(secretBytes[:]); err != nil {
```

Comment:
Make a sized slice, rather than an array you have you slice everywhere you use it